### PR TITLE
Smart maps

### LINKED ISSUE #37: Support new Smart Mapping Renderers
http://blogs.esri.com/esri/arcgis/2015/03/02/introducing-smart-mapping/

I think what got introduced was a bunch of stuff for continuous ramps (size color ect...)

Here is a service that uses a continuous size ramp http://services.arcgis.com/rOo16HdIMeOBI4Mb/arcgis/rest/services/Aggregation%20of%20Trimet%20Transit%20Stops%20to%20Portland%20Neighborhoods/FeatureServer/0?f=pjson

### PR DESCRIPTION
**Pull request by kneemer**
Resolves #37 
Smart Maps integration 
Updated CHANGELOG and release versions
Fixed duplicate drawing of points when doing a point in polygon layer.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,5 +18,6 @@ This change log adheres to standards from [Keep a CHANGELOG](http://keepachangel
 * First Beta release
 * Works with esri-leaflet 1.0.0-rc.4
 
-[Unreleased]: https://github.com/Esri/esri-leaflet-renderers/compare/v0.0.1-beta.2...HEAD
+[Unreleased]: https://github.com/Esri/esri-leaflet-renderers/compare/v0.0.1-beta.3...HEAD
+[0.0.1-beta.2]: https://github.com/Esri/esri-leaflet-renderers/compare/v0.0.1-beta.2...v0.0.1-beta.3
```
Comment (jgravois) [inline CHANGELOG.md]: 0.0.1-beta.3

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,9 +3,13 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 This change log adheres to standards from [Keep a CHANGELOG](http://keepachangelog.com).
 
-## [0.0.1-beta.3][Unreleased] - Unreleased
+## [0.0.1-beta.4][Unreleased] - Unreleased
 
-## [0.0.1-beta.2] - 2015-03-02
+## 0.0.1-beta.3 - 2015-03-24
+* Render from the new visualVariables objects in the renderer JSON
+* Still backwards compatible with classic rendererers
```
Comment (jgravois) [inline CHANGELOG.md]: too many 'erers'

**Comment by jgravois** [comment]
looks great!  at small scales, your renderering of [this](http://www.arcgis.com/home/webmap/viewer.html?url=http://services.arcgis.com/uCXeTVveQzP4IIcx/ArcGIS/rest/services/below-poverty-quarter-mile%20walk%20distance/FeatureServer/0&source=sd) service looks better than the JSAPI

esri leaflet
screenshot 2015-03-24 15 19 58

jsapi
screenshot 2015-03-24 15 19 52

**Comment by kneemer** [comment]
I made the suggested changes

**Pull request merged**